instrumentation/servlet/README.md

```diff
@@ -2,10 +2,10 @@
 
 ## Settings
 
-| System property                                                        | Type    | Default | Description                                         |
-| ---------------------------------------------------------------------- | ------- | ------- | --------------------------------------------------- |
-| `otel.instrumentation.servlet.experimental-span-attributes`            | Boolean | `false` | Enable the capture of experimental span attributes. |
-| `otel.instrumentation.servlet.experimental.capture-request-parameters` | List    | Empty   | Request parameters to be captured (experimental).   |
+| System property                                                        | Type    | Default | Description                                                                      |
+| ---------------------------------------------------------------------- | ------- | ------- |----------------------------------------------------------------------------------|
+| `otel.instrumentation.servlet.experimental-span-attributes`            | Boolean | `false` | Enable the capture of experimental span attributes.                              |
+| `otel.instrumentation.servlet.experimental.capture-request-parameters` | List    | Empty   | Request parameters to be captured (experimental). Use `*` to capture all values. |
```

Comment:
perhaps there should be a warning that using `*` will capture all parameters, including passwords, PII etc. Leaking such sensitive data could get people and organizations in trouble.

Reply:
Imagine putting a secret in a query parameter in 2023. 🙈 
You're right though, probably worth a remark.

Reply:
It is implemented using https://docs.oracle.com/javaee/6/api/javax/servlet/ServletRequest.html#getParameterValues(java.lang.String) which includes all request parameters (POST & GET)

Reply:
The HTTP POST body should not be included, no. It's not a query parameter.

Reply:
> The HTTP POST body should not be included, no. It's not a query parameter.

I'm not sure, I thought the same as @laurit (and that it would include query params embedded in POST body using `x-www-form-urlencoded`)

Reply:
Oh my bad, you're completely right...I had forgotten that HTML form submissions via POST slap the form inputs together via url encoding and put them into the body. I ran a manual test with the agent from this branch and confirmed that both HTML form input values and url query parameters ARE included for POST.

Reply:
As this is a serious footgun WDYT about changing the warning to include relevant samples, such as passwords and credit card numbers, in the warning text to make sure that everybody who intends to use it would not be surprised later on.

Reply:
I have slight reservations about this feature, maybe it would help me to understand the use case better.

@breedx-splk is the PR just based on #9183? (I can ask over there), or do you have other request/need for it?

Reply:
Thanks for checking -- yeah this is purely instigated by #9183. We sometimes have users ask for it, but they probably haven't thought it through..... 😈